test/unit/models/SearchTest.scala

```diff
@@ -99,7 +100,7 @@ class SearchTest extends ModelsBaseSpec {
     }
 
     "Bind populated query string with excessive spaces" in {
-      val extraSpacesParams = populatedParams.mapValues(values => values.map(value => s" $value "))
+      val extraSpacesParams = populatedParams.map { case (fst, snd) => fst -> snd.map(value => s" $value ") }
```

Comment:
Same as above and can be `populatedParams.view.mapValues(values => values.map(value => s" $value ")).toMap`